src/main/java/de/retest/web/RecheckSeleniumAdapter.java

```diff
@@ -22,13 +23,26 @@
 
 import de.retest.recheck.RecheckAdapter;
 import de.retest.ui.DefaultValueFinder;
+import de.retest.ui.descriptors.IdentifyingAttributes;
+import de.retest.ui.descriptors.RetestIdProvider;
 import de.retest.ui.descriptors.RootElement;
 import ru.yandex.qatools.ashot.AShot;
 import ru.yandex.qatools.ashot.shooting.ShootingStrategy;
 import ru.yandex.qatools.ashot.shooting.ViewportPastingDecorator;
 
 public class RecheckSeleniumAdapter implements RecheckAdapter {
 
+	// TODO Remove again after retest 3.0.1 release...
+	public final static RetestIdProvider idProvider = new RetestIdProvider() {
+		@Override
+		public String getRetestId( final IdentifyingAttributes identifyingAttributes ) {
+			return UUID.randomUUID().toString();
+		}
+
+		@Override
+		public void reset() {}
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. ([squid:S1186](https://sonarcloud.io/organizations/retest/rules#rule_key=squid%3AS1186))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.retest%3Arecheck-web&pullRequest=78&issues=AWaLhDVhcPetVqywBZ09&open=AWaLhDVhcPetVqywBZ09)

[ISSUE_KEY]: AWaLhDVhcPetVqywBZ09